sql/mob_pool_mods.sql

```diff
@@ -59,6 +59,7 @@ INSERT INTO `mob_pool_mods` VALUES (276,12,1,1);
 INSERT INTO `mob_pool_mods` VALUES (289,4,4,1);
 INSERT INTO `mob_pool_mods` VALUES (370,28,5,1);
 INSERT INTO `mob_pool_mods` VALUES (410,236,20,0);
+INSERT INTO `mob_pool_mods` VALUES (444,23,23,1); -- Bladmall immune to Sleep, Bind, Silence and Gravity.
```

Comment:
Is this the correct way to do this? I think this needs to be changed.
@TeoTwawki

Reply:
true immunity is a field in mob pools, don't add as a mod.

high resistance will be a mod but only put here if the mob is non-scripted and it affects all mobs using the pool (if its on mob, prolly belongs in the spawn mods if non-scripted)

you can tell apart resist form immunity by the log messages in retail. if it says the word resist, its not immune.